dev/build_prevalidation.py

```diff
@@ -92,6 +92,8 @@ def main():
 
   bom_generator = BomGenerator(options)
   bom_generator.determine_and_tag_versions()
+  bom_generator.determine_and_tag_halyard()
```

Comment:
Last I tried it failed without to know halyard's version in the write_component_version w/o this line. But I might have had a conflicting change - will give it a shot